app/config/config.yml

```diff
@@ -22,7 +22,7 @@ framework:
 twig:
     debug:            '%kernel.debug%'
     strict_variables: '%kernel.debug%'
-    exception_controller: 'FOS\RestBundle\Controller\ExceptionController::showAction'
+    exception_controller: cmf_seo.error.suggestion_provider.controller:listAction
```

Comment:
I propose to use the same strategy the TwigBundle uses: Use `$request->getRequestFormat()` to determine the format. This way, we support all tools in the environment to determine the request format. (e.g. FOSRestBundle, custom things, `_format` route placeholder, etc)